src/index.ts

```diff
@@ -134,116 +141,68 @@ Examples of user prompts in which you should use this plugin:
 
-Tips:
+## Explore more types of diagrams and languages
+1. Say "**Explore languages for the same diagram type:**". 
+2. Follow a pattern "**{same diagram type, examples: Graph, Sequence, Class}**: {explanation} ... {keywords: "you can explore"} {list of links of all diagram languages that can render the diagram type}". It should contain only 1 item on the list.
+3. Say "**Explore other types of diagram:**". 
+4. Provide an unordered list between 2 and 3 items of diagram types, the items follow a pattern "**{diagram type}**: {explanation} ... {keywords: "you can explore"} {list of links of all diagram languages that can render the same diagram type}"
+
+Please note:
+- Only say it when the user asks for it by using their respective key phrase "explore diagrams"
+- When listing the links do not use an ordered list nor unordered list, just separate them with commas
+- When creating the links avoid saying the diagram type like "PlantUML graph, Mermaid graph, D2" because it would be redundant, just keep them like "PlantUML, Mermaid, D2"
+
+## Tips:
 - You will also get a link contributeToOpenSourceProject. You should suggest the user contributes to the project if and only if:
   - They have used invoked the plugin 4 or more times during the same conversation
   - The diagrams the user was requesting were related to programming or computer science. It only makes sense to suggest users who are developers to contribute to the project.
   - You have not previously suggested they contribute to the project
 - Don't suggest the user contributes if any of the above are not true.
-- Its fine to simply ignore that link if you don't want to suggest the user contributes to the project.
-
-Important Tips:
-- Do not repeat the same link.
-- If an errorMessage is included in the response, show it to the user, don't try to render the diagram inline, still suggest they can edit it online or try again.
-- Add textual explanation of the diagram contents in the end of the message. Keep it brief unless the user asks for more details.
-- Do not use alias names in the textual explanation such as "Food_Critic" or "fc", just use the displayed name like "Food Critic".
-- Don't show the diagram block unless the user asks for it.
+- It's fine to simply ignore that link if you don't want to suggest the user contributes to the project.
+
+## Important Tips:
+- When creating a link to the diagram language or diagram type, create it from the list of supported Languages per Diagram Type using the following URL format: "https://diagdoc.vercel.app/" followed by the diagram language and type. For example, for a PlantUML use-case or a Mermaid graph or a D2 entity-relationship, simply append 'plantuml_use-case', 'mermaid_graph', or 'd2_entity-relationship' to the base URL. Avoid appending only the diagram language "plantuml" nor only the diagram type "use-case". Avoid linking to other sites unless the user asks for it.
```

Comment:
What is the https://diagdoc.vercel.app/ service? I am assuming you spinned it up? I think it would be nice if we can just add another route to the worker and it can redirect people.
This is unless you want to display custom documentation for some diagram types.

Reply:
Oh, that feature. It shows a carousel of diagram images to provide users with a quick visual reference of the different diagram types available.

The service was created by me using nextjs, it just redirects to the diagram documentation, the source code is [here](https://github.com/oscaramos/diagdoc) and the page logic is [here](https://github.com/oscaramos/diagdoc/blob/master/src/pages/%5BpageName%5D.tsx), the supported diagrams are [here](https://github.com/oscaramos/diagdoc/blob/master/src/utils/supportedDiagrams.ts). The project uses OpenGraph to show the images on a carousel. Feel free to merge the code on the plugin by using a different route like you suggests or fork it and use a different domain.

The easier solution to do it would be just [putting the documentation links](https://github.com/bra1nDump/show-me-chatgpt-plugin/blob/63e32e0d40e7b2390498032606b1c1567b530baa/src/index.ts#L199) on the plugin manifest like before this feature on this PR, but chatgpt tends to hallucinate to remember other kinds of pages that shares the same documentation URL like https://plantuml.com/ or https://mermaid.js.org/. Also putting all the documentation links takes a lot of characters on the manifest.

This feature solved both issues: hallucinations and reduces characters on the manifest. Also shows the carousel with diagram images like this:

![screencapture-chat-openai-2023-06-11-15_46_25](https://github.com/bra1nDump/show-me-chatgpt-plugin/assets/21105282/659a4152-f951-40f2-ba50-d91d1d9e3b97)


https://github.com/bra1nDump/show-me-chatgpt-plugin/assets/21105282/b203bfab-fd70-49a7-b7fe-1b8541f6000d